src/EventManager.php

```diff
@@ -17,7 +19,7 @@ class EventManager
      *
      * @var object[][]
```

Comment:
should be `array<string, object[]>`. Static analysis must be at a low level since they did not pick it up :thinking:

Reply:
Done.